File: Python/1021.py
num = float(input())+0.001
billetes = [100, 50, 20, 10, 5, 2]
monedas = [1, 0.50, 0.25, 0.10, 0.05, 0.01]

print("NOTAS:")
for i in billetes:
    print('%d nota(s) de R$ %.2f' % (num / i, i))
    num %= i
print("MOEDAS:")
for i in monedas:
    print('%d moeda(s) de R$ %.2f' % (num / i, i))
    num %= i

File: Kotlin/1021.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    var num = input.nextDouble()+ 0.001;
    val billetes = arrayOf(100.0, 50.0, 20.0, 10.0, 5.0, 2.0);
    val monedas = arrayOf(1.00, 0.50, 0.25, 0.10, 0.05, 0.01);
    println("NOTAS:");
    for (i in billetes.indices) {
        println("${(num / billetes[i]).toInt()} nota(s) de R$ %.2f".format(billetes[i]));
        num %= billetes[i];
    }
    println("MOEDAS:");
    for (i in monedas.indices) {
        println("${(num / monedas[i]).toInt()} moeda(s) de R$ %.2f".format(monedas[i]));
        num %= monedas[i];
    }

    
}